README.md

```diff
@@ -89,6 +89,11 @@ The following parameters may be set:
         <td>25080</td>
         <td>This is the port that the Audit Web Service is configured to listen on.</td>
       </tr>
+	  <tr>
+		<td>CAF_LOG_LEVEL</td>
+		<td>INFO</td>
```

Comment:
Should this not be 'info'? i.e. lowercase as you mentioned in the container README this is case sensitive.